app/controllers/api/v1/search_invoice_items_controller.rb

```diff
@@ -10,6 +10,9 @@ def show
   private
 
   def invoice_item_params
+    if params[:unit_price]
+      params[:unit_price] = params[:unit_price].delete(".")
```

Comment:
I like this solution! A good candidate for refactoring maybe but manipulating the params like this is very slick.